lib/system-checker.coffee

```diff
@@ -50,8 +50,13 @@ class SystemChecker
     searchPaths = []
 
     # Windows uses its own API and the paths are unimportant, only attempting
-    # to load it works.
+    # to load it works. We also include OS-specific package instructions to
+    # help solve the problem.
+    packageInstructions = ""
```

Comment:
any reason to not prepopulate this with `"You may need to install the missing language "`?

Reply:
As far as I know, `packageInstructions` will be set for the three platforms Atom runs on because of the `if` checks later. I just didn't know a good generic version for an operating system beyond Windows, Mac, and Linux. So, the blank will have it concat later on safely in the case, but the `if` statements would make any assignment on the initializer unused. I don't feel very strong either way, just felt sufficient to leave it blank.